CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 - `CodeIgniter\Config\Config` is now deprecated in favor of `CodeIgniter\Config\Factories::config()`
 - HTTP Layer Refactor: Numerous deprecations have been made towards a transition to a PSR-compliant HTTP layer. [See the User Guide](user_guide_src/source/installation/upgrade_405.rst)
 
+**Mime Type Detection**
+
+- `Config\Mimes::guessExtensionFromType` now only reverse searches the `$mimes` array if no extension is proposed (i.e., usually not for uploaded files).
+- The fallback values of `UploadedFile->getExtension()` (now: `$this->getClientExtension()`) and `UploadedFile->guessExtension()` (now: `''`) have been changed.
```

Comment:
I don't know if it is just me but can we clarify the `now:` here? I mean does now means the previous behavior, or now means the new behavior made by this change.

Or you can break this into sentences. "The fallback values of `UploadedFile->getExtension()` and `UploadedFile->guessExtension()` have been changed. `UploadedFile->getExtension` now returns blabla from blabla while `UploadedFile->guessExtension` now returns blabla from blabla."

Reply:
Agreed.